internal/services/engines/swift/rules_test.go

```diff
@@ -26,18 +27,378 @@ import (
 func TestRulesVulnerableCode(t *testing.T) {
 	tmpDir := t.TempDir()
 	testcases := []*testutil.RuleTestCase{
+		{
+			Name:     "HS-SWIFT-2",
+			Rule:     NewCoreDataDatabase(),
+			Src:      SampleVulnerableHSSWIFT2,
+			Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-2", ".test")),
+			Findings: []engine.Finding{
+				{
+					CodeSample: `var mainContext: NSManagedObjectContext {`,
+					SourceLocation: engine.Location{
+						Line:     13,
+						Column:   21,
+						Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-2", ".test")),
+					},
+				},
+			},
+		},
+		{
+			Name:     "HS-SWIFT-3",
+			Rule:     NewDTLS12NotUsed(),
+			Src:      SampleVulnerableHSSWIFT3,
+			Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-3", ".test")),
+			Findings: []engine.Finding{
+				{
+					CodeSample: `var tlsMinimumSupportedProtocolVersion: tls_protocol_version_t.DTLSv11`,
+					SourceLocation: engine.Location{
+						Line:     3,
+						Column:   40,
+						Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-3", ".test")),
+					},
+				},
+			},
+		},
+		{
+			Name:     "HS-SWIFT-4",
+			Rule:     NewTLS13NotUsed(),
+			Src:      SampleVulnerableHSSWIFT4,
+			Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-4", ".test")),
+			Findings: []engine.Finding{
+				{
+					CodeSample: `var tlsMinimumSupportedProtocolVersion: tls_protocol_version_t.TLSv11`,
+					SourceLocation: engine.Location{
+						Line:     3,
+						Column:   40,
+						Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-4", ".test")),
+					},
+				},
+			},
+		},
+		{
+			Name:     "HS-SWIFT-5",
+			Rule:     NewReverseEngineering(),
+			Src:      SampleVulnerableHSSWIFT5,
+			Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-5", ".test")),
+			Findings: []engine.Finding{
+				{
+					CodeSample: `.library(name: "FridaGadget", targets: ["FridaGadget"]),`,
+					SourceLocation: engine.Location{
+						Line:     8,
+						Column:   25,
+						Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-5", ".test")),
+					},
+				},
+			},
+		},
 		{
 			Name:     "HS-SWIFT-6",
 			Rule:     NewWeakMD5CryptoCipher(),
 			Src:      SampleVulnerableHSSWIFT6,
-			Filename: filepath.Join(tmpDir, "HS-SWIFT-6"),
+			Filename: filepath.Join(tmpDir, fmt.Sprintf("%s%s", "HS-SWIFT-6", ".test")),
```

Comment:
Why not just use "HS-SWIFT-6.test"?

Reply:
I only Copy and Past from Javascript, Nginx, etc... to keep the standard that exists in the project

Reply:
This fmt.Sprintf make no sense to me. We can just write a raw string. I think that we can write a raw string here and in another pr fix these other tests.

Reply:
Ok, done :)